### PR TITLE
Add GN

### PR DESCRIPTION
**Pull request by zhanghang1989**
## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: python/mxnet/gluon/nn/basic_layers.py [hunk 1]
```diff
@@ -700,3 +701,83 @@ def hybrid_forward(self, F, x, *args):
     def __repr__(self):
         return '{name}({function})'.format(name=self.__class__.__name__,
                                            function=self._func_name)
+
+
+class GropNorm(Block):
```
Comment (szha) [inline python/mxnet/gluon/nn/basic_layers.py]: group norm?

**Comment by Roshrini** [comment]
@zhanghang1989 Thanks for adding GN support. Can you please update PR description with more details? 
@mxnet-label-bot [Gluon, pr-awaiting-review]

**Comment by ankkhedia** [comment]
@zhanghang1989 Could you please look into the CI failures and fix the PR

**Comment by anirudhacharya** [comment]
cc: @codewithsk you were talking of adding GroupNorm operator. 

@zhanghang1989 gentle reminder to update the PR.

**Comment by zhanghang1989** [comment]
Thanks! Will update after CVPR deadline :)

**Comment by vandanavk** [comment]
@mxnet-label-bot update [Gluon, pr-awaiting-response]

@zhanghang1989 Please update the PR description with info about the PR. Please add a test in tests/python/unittest/test_gluon.py

@sandeep-krishnamurthy for review

**Comment by thomelane** [comment]
any update @zhanghang1989? it would be great to use this.

**Comment by zhanghang1989** [comment]
> any update @zhanghang1989? it would be great to use this.
Please see this PR. The module is still not hybridizable. 
https://github.com/apache/incubator-mxnet/pull/14196

**Pull request closed without merging**